nf_core/modules/modules_json.py

```diff
@@ -503,6 +517,16 @@ def check_up_to_date(self):
         if len(subworkflows_missing_from_modules_json) > 0:
             self.resolve_missing_from_modules_json(subworkflows_missing_from_modules_json, "subworkflows")
 
+        # If the "installed" value is not present for modules/subworkflows, add it.
+        for repo, repo_content in self.modules_json["repos"].items():
+            for component_type, dir_content in repo_content.items():
+                for install_dir, installed_components in dir_content.items():
+                    for component, component_features in installed_components.items():
+                        if "installed" not in component_features:
+                            self.modules_json["repos"][repo][component_type][install_dir][component]["installed"] = [
```

Comment:
wouldn't this create a subworkflow entry with the value "subworkflow" in "installed", instead of the subworkflow name?